pkg/config/config.go

```diff
@@ -99,10 +102,6 @@ func GetConfig(client kubernetes.Interface) (*Config, error) {
 }
 
 func (gc *Config) Apply() error {
-	if !gc.populated {
```

Comment:
im curious what was trying to protect before?

Reply:
It was a short-circuit. If the config that came back from the cluster was empty then we skip all of the file / system operations and just return early.